mod/scripts/vscripts/debug_commands.nut

```diff
@@ -142,6 +143,18 @@ execute 'qs --<command>' for more info
 	return true
 }
 
+bool function ClearConsoleWindow( entity player, array<string> args )
+{
+	string msg = ""
+	for( int i; i < 61; i++ )

```

Comment:
Oh, and btw, maybe I should make it so that you can pass the number of lines you want to clear as an optional arg. . .
Opinion?

Reply:
Make another pr idk